packages/inference/src/providers/hf-inference.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * HF-Inference do not have a mapping since all models use IDs from the Hub.
+ *
+ * If you want to try to run inference for a new model locally before it's registered on huggingface.co,
+ * you can add it to the dictionary "HARDCODED_MODEL_ID_MAPPING" in consts.ts, for dev purposes.
+ *
+ * - If you work at Fireworks and want to update this mapping, please use the model mapping API we provide on huggingface.co
+ * - If you're a community member and want to add a new supported HF model to Fireworks, please open an issue on the present repo
+ * and we will tag Fireworks team members.
+ *
+ * Thanks!
+ */
+import { HF_ROUTER_URL } from "../config";
+import type { ProviderConfig, UrlParams, HeaderParams, BodyParams } from "../types";
+
+const makeBody = ({ args, chatCompletion, model }: BodyParams): Record<string, unknown> => {
+	return {
+		...args,
+		...(chatCompletion ? { model } : undefined),
+	};
+};
+
+const makeHeaders = ({ accessToken }: HeaderParams): Record<string, string> => {
+	return { Authorization: `Bearer ${accessToken}` };
+};
+
+const makeUrl = ({ baseUrl, chatCompletion, model, taskHint }: UrlParams): string => {
```

Comment:
I don't think we use this destructuring syntax anywhere else in the package

```suggestion
const makeUrl = (params: UrlParams): string => {
```